client/src/components/Invite/Invite.scss

```diff
@@ -0,0 +1,25 @@
+@import 'styles/variables';
+@import 'styles/helpers';
+
+.Invite {
+  h2 {
+    margin-bottom: $spacing-small;
+  }
+  &__LinkBlock {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: $spacing-small;
+    align-items: center;
+    [type='text'] {
```

Comment:
You can define `input` styles in a global `.css` file (or I can do it if you want).